### ISSUE TITLE
Runtime error archiving 

### ISSUE THREAD

**Issue by fastlane-bot-helper** [fastlane/fastlane#1849]
<table>
      <tr>
        <td>
          <img src='https://avatars0.githubusercontent.com/u/13281667?v=3&s=70' width='35'>
        </td>
        <td>
          Original issue by @gametimebuildserver - Imported from <a href='https://github.com/fastlane/gym/issues/113'>fastlane/gym#113</a>
        </td>
      </tr>
    </table>

```
The following build commands failed:
Libtool /Users/bserver/Library/Developer/Xcode/DerivedData/Gametime-afosxnghsgtxcyajgrkdraetxdxg/Build/Intermediates/ArchiveIntermediates/Beta/IntermediateBuildFilesPath/Pods.build/Beta-iphoneos/Braintree.build/Objects-normal/arm64/libBraintree.a normal arm64
(1 failure)
[17:52:18]: Exit status: 65
/Library/Ruby/Gems/2.0.0/gems/gym-0.8.4/lib/gym/error_handler.rb:52:in `handle_build_error': Error building the application - see the log above (RuntimeError)
	from /Library/Ruby/Gems/2.0.0/gems/gym-0.8.4/lib/gym/runner.rb:80:in `block in build_app'
	from /Library/Ruby/Gems/2.0.0/gems/fastlane_core-0.18.2/lib/fastlane_core/command_executor.rb:51:in `call'
	from /Library/Ruby/Gems/2.0.0/gems/fastlane_core-0.18.2/lib/fastlane_core/command_executor.rb:51:in `execute'
	from /Library/Ruby/Gems/2.0.0/gems/gym-0.8.4/lib/gym/runner.rb:76:in `build_app'
	from /Library/Ruby/Gems/2.0.0/gems/gym-0.8.4/lib/gym/runner.rb:10:in `run'
	from /Library/Ruby/Gems/2.0.0/gems/gym-0.8.4/lib/gym/manager.rb:8:in `work'
	from /Library/Ruby/Gems/2.0.0/gems/gym-0.8.4/lib/gym/commands_generator.rb:41:in `block (2 levels) in run'
	from /Library/Ruby/Gems/2.0.0/gems/commander-4.3.5/lib/commander/command.rb:178:in `call'
	from /Library/Ruby/Gems/2.0.0/gems/commander-4.3.5/lib/commander/command.rb:178:in `call'
	from /Library/Ruby/Gems/2.0.0/gems/commander-4.3.5/lib/commander/command.rb:153:in `run'
	from /Library/Ruby/Gems/2.0.0/gems/commander-4.3.5/lib/commander/runner.rb:428:in `run_active_command'
	from /Library/Ruby/Gems/2.0.0/gems/commander-4.3.5/lib/commander/runner.rb:68:in `run!'
	from /Library/Ruby/Gems/2.0.0/gems/commander-4.3.5/lib/commander/delegates.rb:15:in `run!'
	from /Library/Ruby/Gems/2.0.0/gems/gym-0.8.4/lib/gym/commands_generator.rb:58:in `run'
	from /Library/Ruby/Gems/2.0.0/gems/gym-0.8.4/lib/gym/commands_generator.rb:14:in `start'
	from /Library/Ruby/Gems/2.0.0/gems/gym-0.8.4/bin/gym:6:in `<top (required)>'
	from /usr/local/bin/gym:23:in `load'
	from /usr/local/bin/gym:23:in `<main>'
```

command I was running: 

gym --workspace "projname.xcworkspace" --scheme "Beta" -o $HOME/Desktop/build/ --use_legacy_build_api --clean

**Comment by tomassliz** [comment]
I have the same problem. Once in a while i got `Exit status: 65` but next day is everything ok. This happens when I run Fastlane through Jenkins on our build machine. When I run it directly on build machine everything is ok. I don't understand to this ...

**Comment by NSCabezon** [comment]
@tomassliz In my case was a run script I had, if you do ir on Xcode in the CI machine in the output of the build tells you what failed :)

**Comment by tomassliz** [comment]
I know what failed but I don't know why. This happens only sometimes and in irregular intervals. This is the shortened output:

```
...
▸ Copying Business/StubbedResponses/LoanOffer.xml
▸ Copying Business/StubbedResponses/LoanSubmit.xml
▸ Compiling PanelCollectionCell_FullLink.xib
▸ Compiling AmountTableViewCell.xib
** ARCHIVE FAILED **


The following build commands failed:
CodeSign /Users/hudson/Library/Developer/Xcode/DerivedData/Business-fhkbkcejohabqnahaylrfetrlhdm/Build/Intermediates/ArchiveIntermediates/Individual\ Sit1-Release/IntermediateBuildFilesPath/UninstalledProducts/iphoneos/Tachometer.framework
(1 failure)
[21:07:49]: Exit status: 65
```

**Comment by PeterWong2015** [comment]
I see the error after add asking for permission for sending notifications

**Comment by lakamsani** [comment]
We ran into a similar issue recently with the fastlane suddenly failing on developer laptops with this error towards the last stage of the build. 

==
** ARCHIVE FAILED **
[12:23:31]: Exit status: 65
===

It worked fine for many weeks for our developers.  Our jenkins build server did not see this error with the same fastfile. We investigated potential changes and differences. We added a couple of new pods to our ap recently and we always run Xcode on the developer machines. On the build server we don't run the Xcode App although we do use some of the Xcode command line tools.  Xcode versions are same on both.

The workaround in our case was to drop the Xcode derived data directory on the developer laptops after first exiting Xcode App.

rm -rf /Users/vamsee/Library/Developer/Xcode/DerivedData/Onkore-cqhdpxkmthcxrffvifokbmbpdrxi/

After this the "Exit status: 65" problem went away.

**Comment by TKBurner** [comment]
@lakamsani Thanks for sharing. Glad to hear you got up and runnign 👍

**Comment by slangley** [comment]
I ran into a similar issue today, we are using Crashlytics in a Script Phase, and Jenkins as a runner for the build.  The Crashlytics build phase would through the Archive failed with Exit status: 65 if there was a space in the Jenkins project name.  I removed the space from the project name, and the Exit status went away.

Just putting this here for additional information in case someone else has the same issue.
